requirements/base.txt

```diff
@@ -25,4 +25,4 @@ djangorestframework==3.13.1  # https://github.com/encode/django-rest-framework
 djangorestframework-simplejwt==5.2.0 # https://github.com/SimpleJWT/django-rest-framework-simplejwt
 
 pdfrw==0.4
--e git+https://github.com/deardurham/ciprs-reader.git@main#egg=ciprs-reader
+git+https://github.com/codefordurham/ciprs-reader.git@main#egg=ciprs-reader
```

Comment:
Was this intentional? I don't think we should use the old location here

Reply:
Wow, good catch, how did that happen 😮 -- fixed